_source/_posts/2021-01-27-osgeo-glossary-updated.adoc

```diff
@@ -0,0 +1,56 @@
+---
+layout: custom-post
+title:  "First release of OSGeo Glossary (and how to cite)!"
+date:   2021-01-27 11:30:00 +0800
+categories: update
+---
+
+A new release to celebrate the new year:
+the https://wiki.osgeo.org/wiki/Lexicon_Committee[OSGeo Lexicon Committee]
+has released its first set of terms!
+
+The OSGeo Glossary comprises new and updated terms with definitions
+and examples that are all ready to use.  We have consolidated all
+these terms from OSGeo projects and especially the OSGeo Geo4All
+newsletter.
+
+Our team worked efficiently through the entire process of compiling
+the terms, editing and collating them into a sheet, and then
```

Comment:
Our team worked efficiently through the entire process of compiling, editing, and collating the terms into a single collection and then inserting them into Geolexica (the display framework where the terms were published).